src/utils/scroll.js

```diff
@@ -159,7 +159,8 @@ class ScrollBody {
 
     // These calculations are taken from src/views/canvas/CanvasCoordsCache.js:
     const maxScrollHeight = this.g.zoomer.getMaxAlignmentHeight() - this.g.zoomer.get('alignmentHeight');
-    const maxScrollWidth = this.g.zoomer.getMaxAlignmentWidth() - this.g.zoomer.getAlignmentWidth();
+    // // Note(ritik): Added 1px to fix the border getting clipped for last residue selection in scrollable sequences
+    const maxScrollWidth = this.g.zoomer.getMaxAlignmentWidth() - this.g.zoomer.getAlignmentWidth() + 1;
```

Comment:
For some reason, the border for the last residue in scrollable sequences was being clipped, even after adding 1px to the width.
![image](https://github.com/user-attachments/assets/92ef9039-618c-425f-950f-4718153b11ea)

Added one extra pixel to the max scroll width to fix the same.
![image](https://github.com/user-attachments/assets/d3dfac34-0429-417c-b171-08bb2c709a19)